addressfield.module

```diff
@@ -1211,3 +1304,68 @@ function _addressfield_get_format_callback($handler, $formats) {
     }
   }
 }
+
+/**
+ * Implements hook_webform_component_info().
+ */
+function addressfield_webform_component_info() {
+  $components = array();
+  $components['addressfield'] = array(
+    'label'       => t('Address'),
+    'description' => t('Address field.'),
+    'features'    => array(
+      // Add content to CSV downloads. Defaults to TRUE.
+      'csv'           => TRUE,
+      // Show this component in e-mailed submissions. Defaults to TRUE.
+      'email'         => TRUE,
+      // Allow this component to be used as an e-mail FROM or TO address.
+      // Defaults to FALSE.
+      'email_address' => FALSE,
+      // Allow this component to be used as an e-mail SUBJECT or FROM name.
+      // Defaults to FALSE.
+      'email_name'    => FALSE,
+      // This component may be toggled as required or not. Defaults to TRUE.
+      'required'      => TRUE,
+      // This component has a title that can be toggled as displayed or not.
+      'title_display' => TRUE,
+      // This component has a title that can be displayed inline.
+      'title_inline'  => FALSE,
+      // If this component can be used as a conditional SOURCE. All components
+      // may always be displayed conditionally, regardless of this setting.
+      // Defaults to TRUE.
+      'conditional'   => FALSE,
+      // If this component allows other components to be grouped within it
+      // (like a fieldset or tabs). Defaults to FALSE.
+      'group'         => FALSE,
+      // If this component can be used for SPAM analysis, usually with Mollom.
+      'spam_analysis' => FALSE,
+      // If this component saves a file that can be used as an e-mail
+      // attachment. Defaults to FALSE.
+      'attachment'    => FALSE,
+    ),
+    'file'        => 'addressfield.components.inc',
```

Comment:
I think I would name this file `addressfield.webform.inc` to make it clear that we are supporting the webform module, and not something else like web components.

Reply:
Done!